lib/web_ui/dev/driver_manager.dart

```diff
@@ -137,11 +194,13 @@ abstract class DriverManager {
   static DriverManager chooseDriver(String browser) {
     if (browser == 'chrome') {
       return ChromeDriverManager(browser);
+    } else if (browser == 'firefox') {
+      return FirefoxDriverManager(browser);
     } else if (browser == 'safari' && io.Platform.isMacOS) {
       return SafariDriverManager(browser);
     } else {
-      throw StateError('Integration tests are only supported on Chrome or '
-          'on Safari (running on MacOS)');
+      throw StateError('Integration tests are only supported on Firefox, Chrome'
+          ' or on Safari (running on MacOS)');
```

Comment:
nit: s/or/and/
nit: s/MacOS/macOS/ (throughout the PR)

Reply:
done